presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2584,6 +2584,31 @@ public void testInsertUnsupportedWriteType()
         }
     }
 
+    @Test
+    public void testStagingPartition()
```

Comment:
nit: `testCommitPartitions`?

Reply:
I removed this test and added tests in `testCreate`, `testInsert`, `testInsertIntoExistingPartition` and `testInsertIntoNewPartition`.